components/eventing-controller/pkg/handlers/metrics/collector.go

```diff
@@ -9,16 +9,13 @@ import (
 
 const (
 	// deliveryMetricKey name of the delivery per subscription metric
-	deliveryMetricKey = "delivery_per_subscription"
-
-	// deliveryMetricHelp help text for the delivery per subscription metric
-	deliveryMetricHelp = "Number of dispatched events per subscription"
-
+	deliveryMetricKey = "eventing_ec_delivery_per_subscription_total"
 	// eventTypeSubscribedMetricKey name of the eventType subscribed metric
-	eventTypeSubscribedMetricKey = "event_type_subscribed"
-
+	eventTypeSubscribedMetricKey = "jetstream_ec_event_type_subscribed_total"
```

Comment:
or we could just consistently use `eventing` as the prefix for all metrics since most of the metrics are used mainly for jetstream anyway

Reply:
how about using `eventing` if the metric is used in beb and nats, and `nats` if the metric is only used in nats?